include/freerdp/crypto/crypto.h

```diff
@@ -56,34 +56,50 @@ FREERDP_API CryptoCert crypto_cert_read(BYTE* data, UINT32 length);
 FREERDP_API char* crypto_cert_fingerprint(X509* xcert);
 FREERDP_API char* crypto_cert_subject(X509* xcert);
 FREERDP_API char* crypto_cert_subject_common_name(X509* xcert, int* length);
-FREERDP_API char** crypto_cert_subject_alt_name(X509* xcert, int* count,
-		int** lengths);
-FREERDP_API void crypto_cert_subject_alt_name_free(int count, int *lengths,
-		char** alt_name);
+FREERDP_API char** crypto_cert_get_dns_names(X509* xcert, int* count, int** lengths);
```

Comment:
`size_t` might be the better choice for sizes (since these are new functions)

Reply:
I'd rather keep consistency across those functions, and substitute int -> size_t for lengths in all of them at once, instead of just the new ones.